src/types.rs

```diff
@@ -399,6 +399,16 @@ impl<'a> IntoSkyhashBytes for &'a RawString {
     }
 }
 
+impl FromSkyhashBytes for RawString {
+    fn from_element(element: Element) -> SkyResult<RawString> {
+        let e = match element {
+            Element::Binstr(bstr) => bstr.into(),
+            _ => return Err(Error::ParseError(BAD_ELEMENT.to_owned())),
+        };
+        Ok(e)
+    }
+}
+
```

Comment:
`RawString` is simply a wrapper around `Vec<u8>` and only makes sense for sending queries that contain binary data and not reading them. Is there any specific reason you want to parse into a `RawString` and not a `Vec<u8>`?